index.html

```diff
@@ -9,20 +9,24 @@
 
     <section></section>
     </head>
+    <section><img src="photos/photoNicole.jpeg" alt="Nicole" width="300" height="300" style="border-radius: 50%"></section>
     <section>
-        <h1>ABOUT:</h1>
-        <h2>Hi, my name is Nicole Bradley. I love computer coding. It is my dream to one day be a game developer. I am playing a game called Star Trek Fleet Command. I have to play the games I want to learn how to make. </h2>
+        <h1 style="color:purple;">ABOUT:</h1>
```

Comment:
It's preferable to set up a separate CSS rule to handle this. For example:
```css
h1 {
  color: purple;
}
```
The advantage is that if you need to change it, you can make the change on just one line.